core/metacling/src/TCling.cxx

```diff
@@ -1832,9 +1832,7 @@ void TCling::LoadPCM(std::string pcmFileNameFullPath)
       TMemFile pcmMemFile(RDictFileOpts.c_str(), range);
 
       LoadPCMImpl(pcmMemFile);
-      // FIXME: Uncomment this when we implement UnLoadPCM as per
-      // root-project/root#5420
-      //fPendingRdicts.erase(pendingRdict);
+      fPendingRdicts.erase(pendingRdict);
```

Comment:
This is a bit unrelated to the PR and the comment is still valid.  So I would skip that.

Reply:
It is, if I remove the comments without the changes to TSystem, tests will start to fail. With the change in TSystem uncommenting these lines should work just fine. That code actually detected the issue.

Reply:
Yes.  That is what I expect:

Without this PR, reloading libraries is exercised (a lot).

That the code in TCling.cxx has no negative effect only means that after this PR, the test are no longer reloading libraries that have a PCM (or something like) and thus unloading library and that line are no longer exercise.

Unless I am mistaken it still means that we need to implement UnloadPCM at some point, don't we?

Reply:
Yes, the urgency however changes a bit.